### ISSUE TITLE
the strip has the same color on all the leds

### ISSUE THREAD

**Issue by bfgelectronics** [ahodges9/LedFx#161]
I finished installing this app, but even if I changed all the settings on the left webpage, the strip has the same color on all the less, even if the web interface shows that only specific LEDs should be on. Any suggestion?

**Comment by Mattallmighty** [comment]
Hi @bfgelectronics 
Latest documentation: https://ledfx.readthedocs.io/en/docs/

python version?
LedFx version you are using? 
Have you tried the ledfx.app https://ledfx.app/download/ version 0.0.3
restarted your PC and issue continues?
any changes/deleted your LedFx config file?
What ESP are you using and what software? WLED?

**Comment by bfgelectronics** [comment]
Hi @Mattallmighty , after resetting the wled app from the esp32 and installing the left app, it works! But is there any way I can make the app sync to the computer speakers, not the microphone?

**Comment by Mattallmighty** [comment]
Yeah sure, have a look at: https://ledfx.readthedocs.io/en/docs/#show-me-how-to-make-one - Step 3. Direct computer audio output to LedFx.
